.github/workflows/check-arduino.yml

```diff
@@ -0,0 +1,28 @@
+name: Check Arduino
+
+# See: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run every Tuesday at 8 AM UTC to catch breakage caused by new rules added to Arduino Lint.
+    - cron: "0 8 * * TUE"
+  workflow_dispatch:
+  repository_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Arduino Lint
+        uses: arduino/arduino-lint-action@v1
+        with:
+          compliance: strict
+          library-manager: update
+          # Always use this setting for official repositories. Remove for 3rd party projects.
+          official: false
```

Comment:
This setting is not mentioned in the documentation. I leave it out.

Reply:
I intentionally left it out of the readme because it's only relevant for official Arduino libraries, which have slightly different rules regarding the use of the "Arduino" name (e.g., 3rd party library names must not start with "Arduino", while official library names must start with "Arduino_"). So I felt that mentioning it would add unnecessary complexity to the readme.

It is documented in the action's metadata:
https://github.com/arduino/arduino-lint-action/blob/v1/action.yml#L34

The default value of this input is `false`, so there's no need for adding it to the workflow.

Reply:
Fair enough @mathertel I think I probably copied the config from an actual library, rather than the action example.